docs/src/main.css

```diff
@@ -378,7 +379,6 @@ body._hasBackground {
 .Prose code pre {
   font-family: 'Roboto Mono';
   background-color: transparent;
-  width: 100%;
   display: block;
 }
 
```

Comment:
The 100% width shouldn't be needed as display:block does that for us. Looks good on my local. Can you verify locally?